src/entrypoints/options/views/HistoryView.vue

```
@@ -0,0 +1,176 @@
+
+const logger = createLogger('HistoryView');
+const history = ref<AgentExecutionResult[]>([]);
+
+// option 1
+agentStorage.agentExecutionResults
+  .getValue()
+  .then((value) => {
+    history.value = value;
+  })
+  .catch((error) => {
+    logger.error`Failed to get history, ${error}`;
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)